examples/src/json-codec/common.rs

```diff
@@ -0,0 +1,76 @@
+use bytes::{Buf, BufMut};
```

Comment:
Could also use some module docs here maybe explaining what is going on. Nothing crazy is needed imo